packaging/obs/grafana-ha-cluster-dashboards/grafana-ha-cluster-dashboards.spec

```
@@ -47,6 +56,11 @@ install -Dm644 dashboards/provider-sleha.yaml %{buildroot}%{provisioning_dir}/pr
 %license LICENSE
 %attr(0755,grafana,grafana) %dir %{dashboards_dir}/sleha
 %attr(0644,grafana,grafana) %config %{dashboards_dir}/sleha/*
-%attr(0644,root,root) %config %{provisioning_dir}/provider-sleha.yaml
+%attr(0644,grafana,grafana) %config %{provisioning_dir}/provider-sleha.yaml
```

sorry, I probably wasn't clear with my previous comment, we don't need this change anymore! :)

```suggestion
%attr(0644,root,root) %config %{provisioning_dir}/provider-sleha.yaml
```